lib/AFS/CellCC/Config.pm

```diff
@@ -107,6 +107,8 @@ my @directives = (
     { key => 'restore/check-interval', default => 60, },
     { key => 'restore/queues', type => 'HASH', default => {}, },
     { key => qr:^restore/queues/[^/]+/max-parallel$:, default => 1, },
+    { key => qr:^restore/queues/[^/]+/release/flags$:, type => 'HASH', default => {}, },
+    { key => qr:^restore/queues/[^/]+/release/flags/[^/]+$:, },
```

Comment:
after some offline discussion, I agree with these lines (I was thinking maybe a comment should be added about this... but we already do this for other stuff; I was just being dumb about it). However, it's also possible I think that this could cause problems in the future if someone wants to specify an array or hash value for a vos option. e.g. if there is a CMD_LIST vos parameter that the administrator wants to give, we'll need to give an array in the json, but this will probably fail config checking, since we'll be expecting a SCALAR ref and we'll get an ARRAY ref in the config.

I think I'm just going to not care until that actually becomes a problem, but I just wanted to note it here that that exists.